pkg/ds/daemon_set.go

```diff
@@ -392,6 +413,105 @@ func (ds *daemonSet) unschedule(node types.NodeName) error {
 	return nil
 }
 
+func (ds *daemonSet) PublishToReplication() error {
+	// TODO: We need to specifically turn this on to work
+	if ds.healthChecker == nil {
```

Comment:
I'm keeping this in here for now so that the tests work because I haven't faked the health yet.